addons/sale_project/models/project.py

```diff
@@ -470,7 +470,7 @@ class ProjectTask(models.Model):
         'sale.order.line', 'Sales Order Item',
         copy=False, tracking=True, index='btree_not_null', recursive=True,
         compute='_compute_sale_line', store=True, readonly=False,
-        domain="[('company_id', '=', company_id), ('is_service', '=', True), ('order_partner_id', '=?', partner_id), ('is_expense', '=', False), ('state', 'in', ['sale', 'done'])]",
+        domain="[('company_id', '=', company_id), ('is_service', '=', True),  '|', ('order_partner_id', '=?', partner_id), ('order_id.partner_shipping_id', '=?', partner_id), ('is_expense', '=', False), ('state', 'in', ['sale', 'done'])]",
```

Comment:
I think this should move into industry_fsm_sale right?

Reply:
@IT-Ideas   @xavierbol  was suggested to do it directly in sale_project.  https://github.com/odoo/enterprise/pull/25692
Thanks